doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -433,6 +433,17 @@ implementation, the tactic will always choose the first possible
 solution to the set of constraints generated by a rewrite and will not
 try to find *all* the possible solutions to warn the user about them.
 
+Compatibility with ssreflect's rewrite
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When calling ssreflect's :tacn:`rewrite` tactic with a lemma, a check is
+performed to see if the rewriting relation can be handled by generalized rewriting.
+This relies on the ``RewriteRelation A R`` typeclass, indexed by a type and relation,
+that registers relations handled by generalized rewriting.
```

Comment:
```suggestion
The :tacn:`rewrite (ssreflect)` tactic uses generalized rewriting when possible.
The ``RewriteRelation A R`` typeclass, indexed by a type and relation, identifies
the relations that generalized rewriting handles.
```

If that says what's needed here.

If some of this information applies only to the one tactic, maybe it should appear in the description of `rewrite` in the SSR chapter?

Reply:
We could backlink from the ssr chapter indeed.